doc/developer_onboarding.md

```diff
@@ -13,6 +13,10 @@ Create a [new issue](https://github.com/18F/C2/issues/new), and copy the raw Mar
 * [ ] Subscribe to the project on CircleCI
     1. [Authorize With GitHub](https://circleci.com/signup)
     1. Go to [the project page](https://circleci.com/gh/18F/C2) and "follow the C2 project"
+* [ ] Access support emails `capdevs@gsa.gov` and `communicart.sender@gsa.gov`
+    1. Make sure someone has requested/completed access for you (See tasks for @oldmember)
+    1. Go to Gmail and click 'Add Account'. Enter in each of these email addresses. If you are asked for a password, leave it blank. As long as you are already logged into your GSA email, the email addresses should be loaded.
+    1. One would expect the newly added email to load automatically, but it doesn't. To load the new email, simply click on your email address at the top right of the page to show all of your loaded email accounts. Click on the newly added email address (capdevs or communicart.sender) to load it.
```

Comment:
This definitely seems like a "later" thing...mind moving it to the next section?